webonary-cloud-api/lambda/getDictionary.ts

```diff
@@ -105,6 +106,41 @@ export async function handler(event: APIGatewayEvent): Promise<APIGatewayProxyRe
   );
   dbItem.definitionOrGlossLangs = [...new Set(senseLangs.flat(1))].filter((lang) => lang !== '');
 
+  // get parts of speech counts
+  dbItem.partsOfSpeech =
+    dbItem.partsOfSpeech
+      ?.filter(
+        // de-dup
+        (part, index, self) =>
+          index ===
+          self.findIndex((p) => (p.lang === part.lang && p.abbreviation) === part.abbreviation),
+      )
+      ?.map((part) => {
+        // For some reason, FLex sends these decomposed, but entries are composed (e.g. for accented chars)
+        return { ...part, abbreviation: part.abbreviation.normalize('NFC') };
+      }) ?? [];
+
+  const partsOfSpeechCounts = await Promise.all(
+    dbItem.partsOfSpeech.map(async ({ abbreviation }) => {
```

Comment:
yes, will do that in the next PR, along with other summary data like this.
this will require a data migration script, so takes a bit more work and timing. 
did not want to do another data migration so soon, after having done a major one just recently....

thanks!